Update changelog for v1.5.0 and v1.5.1 (#493)
Kong/kubernetes-configuration: CHANGELOG.md

  [#441](https://github.com/Kong/kubernetes-configuration/pull/441)
  [#454](https://github.com/Kong/kubernetes-configuration/pull/454)

## [v1.5.1]

[v1.5.1]: https://github.com/Kong/kubernetes-configuration/compare/v1.5.0...v1.5.1

### Cleanups

> - By updating the VERSION and regenerating the manifests, it ensures that the metadata of the CRDs remains consistent and up-to-date.

Reviewer: ```suggestion
- Update the CRDs metadata with up to date version.
```

How about this?

Author: LGTM
